nixos/modules/tasks/kbd.nix

```diff
@@ -51,9 +51,10 @@ in
     (mkIf setVconsole {
       environment.systemPackages = [ pkgs.kbd ];
 
+      environment.pathsToLink = [ "/share/consolefonts" "/share/keymaps" ];
+
       # Let systemd-vconsole-setup.service do the work of setting up the
-      # virtual consoles.  FIXME: trigger a restart of
-      # systemd-vconsole-setup.service if /etc/vconsole.conf changes.
+      # virtual consoles.
```

Comment:
Good idea, I'll split it (and add a missing patch file) after additional testing.